keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -324,85 +396,147 @@ Contains a unique identifier for the containing cluster.
+    potential changes if they live beyond the ClusterSet membership.
+*   **May** rely on the existence of an identifier for clusters that do not
+    belong to a ClusterSet so long as the implementation provides one.
 
 
 ##### Notable scenarios
 
-**Renaming a cluster**: Since a `cluster.clusterset.k8s.io ClusterProperty` must be immutable for the duration of its *membership* in a given ClusterSet, the property contents can be "changed" by unregistering the cluster from the ClusterSet and reregistering it with the new name.
-
-**Reusing cluster names**: Since an `cluster.clusterset.k8s.io ClusterProperty` has no restrictions on whether or not a ClusterProperty can be repeatable, if a cluster unregisters from a ClusterSet it is permitted under this standard to rejoin later with the same `cluster.clusterset.k8s.io ClusterProperty` it had before. Similarly, a *different* cluster could join a ClusterSet with the same `cluster.clusterset.k8s.io ClusterProperty` that had been used by another cluster previously, as long as both do not have membership in the same ClusterSet at the same time. Finally, two or more clusters may have the same `cluster.clusterset.k8s.io ClusterProperty` concurrently (though they **should** not; see "Uniqueness" above) *as long as* they both do not have membership in the same ClusterSet.
+**Reusing cluster names**: Since an `cluster.clusterset.k8s.io ClusterProperty`
+has no restrictions on whether or not a ClusterProperty can be repeatable, if a
+cluster unregisters from a ClusterSet it is permitted under this standard to
+rejoin later with the same `cluster.clusterset.k8s.io ClusterProperty` it had
+before. Similarly, a *different* cluster could join a ClusterSet with the same
+`cluster.clusterset.k8s.io ClusterProperty` that had been used by another
+cluster previously, as long as both do not have membership in the same
+ClusterSet at the same time. Finally, two or more clusters may have the same
+`cluster.clusterset.k8s.io ClusterProperty` concurrently (though they **should**
+not; see "Uniqueness" above) *as long as* they both do not have membership in
+the same ClusterSet.
 
 **Cluster identifier composed of two DNS labels**: If a clusterset includes
 clusters registered with multiple cluster registries, the value of
 `cluster.clusterset.k8s.io ClusterProperty` may not uniquely identify a cluster
-within the clusterset. In such case the value of `cluster.clusterset.k8s.io ClusterProperty`
-may be composed of two DNS labels separated with a dot. The additional DNS label
-is implementation-dependent and allows the implementation to uniquely identify
-the cluster registry with which a cluster is registered.
+within the clusterset. In such case the value of `cluster.clusterset.k8s.io
+ClusterProperty` may be composed of two DNS labels separated with a dot. The
+additional DNS label is implementation-dependent and allows the implementation
+to uniquely identify the cluster registry with which a cluster is registered.
```

Comment:
Does this need to be updated to reflect the changes in the Contents section from multiple DNS labels to a DNS subdomain?

Reply:
Good catch. Removed this notable scenario, as now that we use a subdomain it is not notable :)